src/reference/90-Developers-Guide/01-Modularization/01-Module-Summary.md

```diff
@@ -33,21 +33,21 @@ The responsibility of the serialization API is to turn values into JSON.
 
 #### Util APIs ([sbt/util][utilrepo])
 
-Util APIs provide commonly used features like logging and internal datatypes used by sbt.
+Util APIs provide commonly used features like logging and internal data types used by sbt.
```

Comment:
Yeah at first I just made Vim spellcheck happy because I don't have an opinion, but I yielded after getting to the section on `sbt-datatype`. It's obviously in common use in industry vernacular so I'm happy to revert these to stay consistent.